en/docs/reference/synapse-properties/expressions.md

```diff
@@ -230,271 +234,303 @@ Syntax:
 -   url-encode(string value)
 -   url-encode(string value, string charset)
 
-## Synapse XPath Variables 
+### Synapse XPath variables
 
 There is a set of predefined XPath variables that you can directly use
 to write XPaths in the Synapse configuration, instead of using the
 synapse:get-property() function . These XPath variables get properties
 of various scopes as follows:
 
-### $body
+#### $ctx
 
-The SOAP 1.1 or 1.2 body element. For example, the expression `$body//getQuote` refers to the first `getQuote` element in a SOAP body, regardless of whether the message is SOAP-11 or SOAP-12. We have discussed an example below.
+Prefix for Synapse MessageContext properties and gets a property at the default scope. For example, to get the value of Synapse message context property with name `ERROR_MESSAGE`, use the XPath expression `$ctx:ERROR_MESSAGE`. We have discussed an example below.
 
-**Example of $body usage**:
+**Example of $ctx usage**:
```

Comment:
This `Example of $ctx usage` header is bolder than `$ctx`. Shall we fix that? Please update the rest as well.

Reply:
$ctx is a header and Example of $ctx usage is bold (not a header).

Reply:
Yeah.but ideally `Example of $ctx usage` should look like a sub section under `$ctx`. Currently it looks more highlighted as follows,

<img width="604" alt="Screenshot 2024-08-13 at 09 01 01" src="https://github.com/user-attachments/assets/9cc41544-e8ed-4d1b-8c8f-4b25500d2fee">